src/px-script/utils/index.js

```diff
@@ -9,7 +9,18 @@ export const findAncestor = (el, cls) => {
     return el;
 };
 
+export const isWithinBoundingBox = (x, y, element) => {
```

Comment:
How much better is this than checking if an element simply contains target?

Reply:
Well, for this component, that would suffice. But sometimes (like in the topbar menu) there are several elements that not necessarily belong to each other that should prevent (or enable) an event. It was used here because the function was already created.